as2/6.0/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc

```diff
@@ -9,53 +11,51 @@
 |5.x |6.x
 |===
 
-== Changes in This Release (v6.0)
-
-This release contains the following changes:
+== Changes in This Release 
 
-* Digitally signed copies of outgoing and received messages and their corresponding receipts can be saved for non-repudiation purposes as long as they are configured and enabled on AS2 connector operations and sources.
+AS2 Connector 6.0 contains the following changes:
 
-* The new Mule source `Non repudiation listener` must be user-configured to persist the non-Repudiation information provided by any of the AS2 Connector
+* Digitally signed copies of outgoing and received messages and their corresponding receipts can be saved for non-repudiation purposes as long as they are configured and enabled on AS2 Connector operations and sources.
+* The new Mule source `Non repudiation listener` must be user-configured to persist the non-repudiation information provided by any of the AS2 Connector
```

Comment:
```suggestion
* The new source `Non repudiation listener` must be user configured to persist the nonrepudiation information provided by any of the AS2 Connector
```

Reply:
I haven't found examples of user-configured, but user-defined exists. So feel free to take it or leave it.